source/portable/NetworkInterface/Common/phyHandling.c

```diff
@@ -613,6 +613,7 @@ BaseType_t xPhyStartAutoNegotiation( EthernetPhy_t * pxPhyObject,
         {
             BaseType_t xPhyAddress = pxPhyObject->ucPhyIndexes[ xPhyIndex ];
             uint32_t ulPhyID = pxPhyObject->ulPhyIDs[ xPhyIndex ];
+            uint32_t ulPHYLinkStatus;
```

Comment:
It is good to place it here, but please initialise it to zero:
~~~diff
- uint32_t ulPHYLinkStatus;
+ uint32_t ulPHYLinkStatus = 0U;
~~~
and don't clear it later on.